packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.story.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean, object } from '@storybook/addon-knobs';
+import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
+import { ButtonCascader } from './ButtonCascader';
+
+const getKnobs = () => {
+  return {
+    disabled: boolean('Disabled', false),
+    text: text('Button Text', 'Click me!'),
+    options: object('Options', [
+      {
+        label: 'A',
+        value: 'A',
+        children: [
+          { label: 'B', value: 'B' },
+          { label: 'C', value: 'C' },
+        ],
+      },
+      { label: 'D', value: 'D' },
+    ]),
+  };
+};
+
+const ButtonCascaderStories = storiesOf('UI/ButtonCascader', module);
```

Comment:
Please use CSF(https://storybook.js.org/docs/formats/component-story-format/) for stories definition